core/core.mk

```diff
@@ -88,9 +91,17 @@ libdir = lib/libmpa
 include mk/lib.mk
 base-prefix :=
 
+ifeq ($(WITH_CRYPTO), tomcrypt)
 libname = tomcrypt
 libdir = core/lib/libtomcrypt
 include mk/lib.mk
+else ifeq ($(WITH_CRYPTO), platcrypt)
+libname = platcrypt
+libdir = core/arch/arm/plat-$(PLATFORM)/lib/libplatcrypt
```

Comment:
I'm not super happy about this. How about something like:
```
CRYPTOLIB_NAME ?= tomcrypt
CRYPTOLIB_DIR ?= core/lib/libtomcrypt
libname = CRYPTOLIB_NAME
libdir = CRYPTOLIB_DIR
...
```
instead?

@jforissier do you have any input?

Reply:
Agreed. Add a `CFG_` prefix and add the missing `$(...)` and that should be perfect ;-)
```
CFG_CRYPTOLIB_NAME ?= tomcrypt
CFG_CRYPTOLIB_DIR ?= core/lib/libtomcrypt
libname = $(CFG_CRYPTOLIB_NAME)
libdir = $(CFG_CRYPTOLIB_DIR)
```

@deebee-v2 would that work for your use case?

Reply:
Yes that works perfectly, thanks!